students/wachulec_maria/lesson_03_functions/calculator.py

```diff
@@ -0,0 +1,74 @@
+def show_available_options():
```

Comment:
There is no docstrings in the whole file. Please add them.
https://www.python.org/dev/peps/pep-0257/